.github/workflows/deploy_latest_release.yml

```diff
@@ -58,3 +70,43 @@ jobs:
           PROJECT: pwa-kit
           MESSAGE: "tag ${{ github.event.release.tag_name }}"
```

Comment:
Can we add a more useful message to the bundle?
<img width="1125" alt="Screenshot 2025-03-11 at 4 19 48 PM" src="https://github.com/user-attachments/assets/252a3971-f4dd-4a84-85da-c6e344ba6b78" />

Right now looks like the value for `github.event.release.tag_name` gets lots, so we end up just with "tag".